jwst/lib/pipe_utils.py

```diff
@@ -3,10 +3,6 @@
 import numpy as np
 
 from ..associations.lib.dms_base import TSO_EXP_TYPES
-from ..datamodels import CubeModel
-
-# Model types that typically represent TSO's
-TSO_MODEL_TYPES = (CubeModel,)
```

Comment:
One of the tests in
```
jwst/lib/tests/test_pipe_utils.py
```
tests this.  You'll want to modify or remove the test.

Reply:
Done (in latest commit)